scripts/eessi-upload-to-staging

```diff
@@ -185,6 +213,21 @@ done
 # restore potentially parsed filename(s) into $*
 set -- "${POSITIONAL_ARGS[@]}"
 
+# ensure that either none or both of $sign_key and $sign_script are defined
+if [[ -n "${sign_key}" ]] && [[ -n "${sign_script}" ]]; then
+  sign=1
+elif [[ -n "${sign_key}" ]]; then
+  sign=0
```

Comment:
Nitpicking, can maybe be tackled in a follow-up PR: I would've used `'no'` and `'yes'` here, to avoid confusion.

`0` is often a sign of success in bash scripts, here's it's a negative value, `'no'` leaves no room for confusion